src/Admin/Admin.php

```diff
@@ -261,6 +263,22 @@ protected function enableReports(): bool {
 		return apply_filters( 'woocommerce_gla_enable_reports', true );
 	}
 
+	/**
+	 * Add suggested privacy policy content
+	 *
+	 * @return void
+	 */
+	public function privacy_policy() {
+		$content = '<p class="privacy-policy-tutorial">' . sprintf(
```

Comment:
I believe that adding the class `privacy-policy-tutorial` might cause the `Copy suggested policy text to clipboard` button to not function correctly. According to the [wp_add_privacy_policy_content](https://developer.wordpress.org/reference/functions/wp_add_privacy_policy_content/#description) doc:

> Any content contained within HTML elements that have the .privacy-policy-tutorial CSS class applied will be omitted from the clipboard when the section content is copied.